docs/add-ons/crowdstrike-falcon.md

```diff
@@ -0,0 +1,46 @@
+# CrowdStrike Falcon
+
+`falcon` is a terraform module that can automate the deployment of CrowdStrike Falcon Sensor and the Kubernetes Protection Agent on a Kubernetes cluster.
+
+## Falcon Operator
+Falcon Operator is a Kubernetes operator that manages the deployment of the CrowdStrike Falcon Sensor on a Kubernetes cluster. The CrowdStrike Falcon Sensor provides runtime protection for workloads running on a Kubernetes cluster.
+
+More information can be found in the [Operator submodule](https://github.com/CrowdStrike/terraform-kubectl-falcon/blob/main/modules/operator/README.md).
+
+## Kubernetes Protection Agent (KPA)
+
+The Kubernetes Protection Agent provides visibility into the cluster by collecting event information from the Kubernetes layer. These events are correlated to sensor events and cloud events to provide complete cluster visibility.
+
+More information can be found in the [KPA submodule](https://github.com/CrowdStrike/terraform-kubectl-falcon/blob/main/modules/k8s-protection-agent/README.md).
+
+## Usage
+
+Refer to the [`falcon module`](https://github.com/CrowdStrike/terraform-kubectl-falcon) documentation for the most up-to-date information on inputs and outputs.
+
+## Example
+
+You will call the `falcon` module similar to how you would call any other terraform module. The [module documentation](https://github.com/CrowdStrike/terraform-kubectl-falcon) provides a list of all the inputs and outputs.
+
+```hcl
+#---------------------------------------------------------------
```

Comment:
@ffalor have you tested this example to make sure it is compatible in that mode?

Reply:
let me modify the example from my previous PR and I'll add it to my module repo as a full E2E example of how to use this with blueprints. I'll comment back once I have and you can use the same example on your end.

Reply:
<img width="857" alt="image" src="https://user-images.githubusercontent.com/35144141/223565819-d880bf29-6f65-4727-9de4-737e53b3695e.png">


the example code is here: https://github.com/CrowdStrike/terraform-kubectl-falcon/tree/v0.1.0/examples/aws-eks-blueprint-example

Reply:
I've never used the EKS blueprints repo so lmk if I missed something, but my example is based on another example in this repo.

Reply:
@ffalor your examples states "Click Self-Managed Kubernetes Service". However, I also see AWS EKS in the list, which seems to be a more natural choice. Will that path work for the customers? Also when doing it at scale, is there a way to reuse a secret across a fleet of clusters (e.g. all production clusters in the account across all regions)?  One of the goals of the blueprint is to eliminate any per-cluster configuration and make fully automatic.

Reply:
the example provided here is not valid - lets just remove and use the linked example on line 24 (tl;dr - line 26+ can be deleted)

Reply:
@shapirov103 the only reason the instructions on generating a "Self-Managed Kubernetes Services" exists is to get access to the DockerApiToken that is used to pull the container images that are required by the helm chart that is deployed. If you follow the instructions for registering AWS EKS clusters that you saw it would generate scripts that would deploy a CFT template. This is different from what we are doing here using helm + terraform.

The same DockerAPIToken can be used for any number of clusters. It is just a token that is used to pull the container images from our internal registry.

Let me know if I need to clarify anything.

Reply:
@bryantbiggs the example on line 26+ is the exact same example that is linked just without the required/optional parameters. The goal was to show how you would call our module just like calling the EKS blueprint modules. I was told it would be best to not to include anything that could change.

If we are okay with including input variables i'd be happy to update the example to include only the required input variables that I know will be unlikely to ever change, or if we just think its redundant and we should just link to the full example only that works for me.

Reply:
@ffalor understood about the DockerAPIToken. What about cid? Is it something that can be generated from the blueprint? 
Use case: need to provision 20 clusters in prod across multiple regions. How do I feed cids? Will it mean accessing your dashboard 20 times and generating a unique cid?

Reply:
A customer will only have a single CID. Some customers may have more than one, but that is rare. CIDs do not change either.